src/mame/sega/segaai.cpp

```diff
@@ -0,0 +1,822 @@
+  lowered and raised the volume."
+
+===========================================================================
+
+ Sega AI Computer quick PCB overview by Chris Covell
+
+ Major ICs
+
+ IC 1    D701080-5     (86/09?)  NEC V20 CPU       DIP40
+ IC 2    315-5200      (86/25)   SEGA          QFP100
+ IC 3    27C512-25     (86/15)   64K EPROM "E000  8/24"
+ IC 4    27C512-25     (86/06)   64K EPROM "F000  7/21"
+ IC 5    MPR-7689      (86/22)   SEGA "264 AA E79" (ROM) DIP28
+ IC 10   V9938                   Yamaha MSX2 VDP
+ IC 13   D7759C        (86/12)   NEC Speech Synthesizer   DIP40
+ IC 14   MPR-7619      (86/23)   SEGA (ROM)      DIP28
+ IC 15   MPR-7620      (86/23)   SEGA (ROM)      DIP28
+ IC 16   SN76489AN               TI PSG         DIP16
+ IC 17   D8251AFC      (86/09)   NEC Communications Interface DIP28
+ IC 18   315-5201      (86/25)   SEGA (bodge wire on pins 9,10) DIP20
+ IC 19   M5204A        (87?/01)  OKI
+ IC 20   D8255AC-2     (86/08)   NEC Peripheral Interface DIP40
+
+ IC 6,7,8,9,11,12   D41464C-12   NEC 32K DRAMs - 128K RAM, 64K VRAM
+
+ Crystals, etc
+
+ X1   20.000           "KDS 6D"
+ X2   21.47727         "KDS"
+ X3   640kHz           "CSB 640 P"
+
+ Connectors
+
+ CN1   6-pin DIN Power connector
+ CN2   8-pin DIN "AUX" connector
+ CN3   Video phono jack
+ CN4   Audio phono jack
+ CN5   35-pin Sega MyCard connector
+ CN6   60-pin expansion connector A1..A30 Bottom, B1..B30 Top
+ CN7   9-pin header connector to "Joy, Button, LED" unit
+ CN8   13(?) pin flat flex connector to pressure pad
+ CN9   9-pin header connector to tape drive motor, etc.
+ CN10   13-pin header connector to tape heads
+ JP2   2-wire header to SW2 button board
+ PJ1   7-wire header to Keyboard / Mic connector board
+ MIC   2-wire header to mic on KB/Mic board
+ SW1   Reset Switch
+
+ Power switch is on the AC Adaptor
+
+ Joypad unit (by Mitsumi) has U/D/L/R, "PL" and "PR" buttons, and a power LED.
+
+Power Connector Pinout (Seen from AC Adaptor plug):
+   1     5        1  12V COM    5   5V COM
+      6           2  12V OUT    6   5V OUT
+   2     4        3   5V COM
+      3           4   5V OUT
+
+AUX Connector Pinout:
+   7   6          1 +5V(?)      5 csync
+  3  8  1         2 GND         6 green
+   5   4          3 blue        7 Audio out
+     2            4 +5V(?)      8 red
+
+New JIS Keyboard Connector Pinout:
+    1 2           1,2,3 data lines
+  3 4   5         4 ??          5,8 data lines
+   6 7 8          6 GND         7 +5V
+
+
+*/
+
+#include "emu.h"
+
+#include "bus/segaai/segaai_exp.h"
+#include "bus/segaai/segaai_slot.h"
+#include "cpu/nec/nec.h"
+#include "imagedev/cassette.h"
+#include "machine/i8251.h"
+#include "machine/i8255.h"
+#include "sound/sn76496.h"
+#include "sound/upd7759.h"
+#include "video/v9938.h"
+
+#include "softlist.h"
+#include "softlist_dev.h"
+#include "speaker.h"
+
+#include "segaai.lh"
+
+#define VERBOSE (LOG_GENERAL)
+#include "logmacro.h"
```

Comment:
Should verbose logging be enabled?

Reply:
No, it shouldn't. Disabled the verbose logging.